.github/workflows/ci.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
     - uses: actions/setup-go@v4
       with:
         go-version: '1.19.2'
+        repo-token: ${{ secrets.YOSHI_CODE_BOT_TOKEN }}
```

Comment:
Oops, I've removed it. I was initially running into errors with cloning `googleapis/googleapis`, so I was trying out using a token, but it turns out that replacing the submodule URL with a `https://` - url instead of `git@github` did the tric.